### PR TITLE
Dev-site Examples has been updated with realistic examples

### PR DESCRIPTION
**Pull request by KV106606Viswanath**
### Summary
UXPLATFORM-8785 - Updated component examples content

### Solution
Updated the component examples, Replaced Lorem Ipsum and non meaningful text with meaningful text on terra-framework components.

### CODE REVIEW

**Comment by sdadn** [comment]
Can you update the PR title to be more descriptive?

**Comment by supreethmr** [comment]
Can you add list components that you modified on Jira !!

**Comment by KV106606Viswanath** [comment]
> Can you add list components that you modified on Jira !!

1. application-layout
2. application-utility
3. notification-dialog
4. tabs

**Review by supreethmr — APPROVED** [review]

**Comment by KV106606Viswanath** [comment]
> Can you add list components that you modified on Jira !!

Component list has been updated in JIRA

File: packages/terra-framework-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated component examples.
```
Comment (sycombs) [inline packages/terra-framework-docs/CHANGELOG.md:6]:
```suggestion
  * Updated `notification-dialog` and `tabs` examples.
```

Comment (KV106606Viswanath) [reply packages/terra-framework-docs/CHANGELOG.md]: Changes updated

**Review by sycombs — APPROVED** [review]

**Pull request merged**